src/schema/artwork/index.ts

```diff
@@ -526,6 +529,7 @@ export const artworkFields = () => {
         _request,
         { rootValue: { partnerLoader } }
       ) => {
+        // Used for Eigen compatibility, see converation at: https://github.com/artsy/metaphysics/pull/1350
```

Comment:
Same, would you mind moving the comment back. This one seems important.

Reply:
Oh.. Bummer. I did not even notice. One min.